packages/components/src/radio-group/index.ts

```diff
@@ -32,15 +38,60 @@ function updateValueAction<T>(
   });
 }
 
+function updateUiAction<T>(
```

Comment:
Question: Suggestion: Minor: Should we rename updateUiAction to updateUIAction or maybe to updateRootContainerAction because UI is very common and doesn't provide necessary specifics?

Reply:
Renamed this action :)